src/Windows/Avalonia.Win32/OpenGl/WglDisplay.cs

```diff
@@ -126,15 +126,20 @@ private static void DebugCallback(int source, int type, int id, int severity, in
                     IntPtr context;
                     using (shareContext?.Lock())
                     {
+                        var profileMask = WGL_CONTEXT_CORE_PROFILE_BIT_ARB;
+                        if (version.EnableCompatibilityProfile &&
+                            version.Major * 10 + version.Minor >= 32)
```

Comment:
That's not a proper way to compare versions. Consider using System.Version struct if you don't want to implement the logic manually.

Reply:
Ah... yes, I copied/pasted this from some C code; seemed clever, but indeed, it only works because there was never a version 2.12 of OpenGL...